arviz/tests/test_plots.py

```diff
@@ -80,8 +81,10 @@ def test_pairplot(self):
         pairplot(self.short_trace, kind='hexbin', var_names=['theta'],
                  coords={'theta_dim_0': [0, 1]}, plot_kwargs={'cmap': 'viridis'}, textsize=20)
 
-    def test_ppcplot(self):
-        ppcplot(self.data['y'], self.sample_ppc)
+    @pytest.mark.parametrize('kind', ['kde', 'cumulative'])
+    def test_ppcplot(self, kind):
+        data = pymc3_to_inference_data(trace=self.short_trace, posterior_predictive=self.sample_ppc)
+        ppcplot(data, kind=kind)
```

Comment:
Related to the question above. It seems that PPC plot presents an inconsistent interface to the other plots because the user needs to convert to an InferenceData object themselves, whereas in the other plots it happens under the hood. Is my understanding correct?

Reply:
My guess is that we can't infer what are the predictive values needed for comparisons.

This plot is not trivial, because user needs to do extra work to get it working.